examples/blood_query_mirror.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
```

Comment:
I understad that the code in this PR extends the functionality of "update attribute action", so documentation in https://github.com/telefonicaid/perseo-fe/blob/master/documentation/plain_rules.md#update-attribute-action should be modified.

In particular, appart of the new fiels for entity id and type, the templates lenguage (based on `${..}` if I'm remembering correctly) should be described.